### PR TITLE
ENG-33 redirect after login

### PR DESCRIPTION
**Pull request by jeffgo10**
made for both default and keycloak auth

### CODE REVIEW

**Comment by entando-service** [comment]
All is well

File: src/auth/default/DefaultAuthProvider.js [hunk 1]
```diff
@@ -1,13 +1,24 @@
...
+import { history, ROUTE_HOME, ROUTE_DASHBOARD } from 'app-init/router';
 
 const DefaultAuthProvider = ({ children }) => {
   const auth = {
     enabled: false,
     authenticated: false,
-    logout: () => history.push(ROUTE_HOME),
+    logout: (status) => {
+      let addredirect = '';
+      if (status) {
```
Comment (gidesan) [inline src/auth/default/DefaultAuthProvider.js]: I'd suggest to use const and binary operator (extracting some logic to a new function to ease the process)

Comment (jeffgo10) [reply src/auth/default/DefaultAuthProvider.js]: done

**Comment by entando-service** [comment]
All is well

**Comment by jeffgo10** [comment]
apimanager now bumped to 4.3.0 and tested. pls approve @gidesan

**Comment by entando-service** [comment]
All is well

**Pull request merged**